public/index.html

```diff
@@ -82,10 +82,7 @@ <h2>das ist die unterueberschrift</h2>
             <p>
                 <code>das hier ist der code</code>
             </p>
-            <video src="Demofilm.mp4" controls>
-                <br>
-                <a href="https://www.youtube.com/watch?v=FYH9n37B7Yw"></a>
-            </video>
+            <video src="videos/demofilm.mp4" controls></video>
```

Comment:
Please use CSS to resize with of video to `300px`

Reply:
Ich hätte die größe mit height="size-300" width="size-300" verändert aber das funktioniert nicht